tests/contrib/gunicorn/wsgi_mw_app.py

```diff
@@ -1,8 +1,11 @@
 from ddtrace import tracer
 from ddtrace.contrib.wsgi import DDWSGIMiddleware
+from ddtrace.debugging import DynamicInstrumentation
 from tests.webclient import PingFilter
 
 
+DynamicInstrumentation.enable()
```

Comment:
@Yun-Kim can you help me understand why you added this to the test app? I think it could play an important role in the tests I'm writing and want to check my understanding. Thanks!

Reply:
@emmettbutler good question! I added Dynamic Instrumentation here while tinkering around with the test scenarios to see if we could see any unexpected behavior. If I recall correctly @P403n1x87 mentioned there were known issues with Dynamic Instrumentation with gevent, although I'm not sure if it was enabling Dynamic instrumentation alone that caused the issues.

Reply:
I think the idea was to start a service that would register after-fork hooks that re-start background threads. DI is an example where this happens, since it requires RCM and the uploader thread to be started soon after fork. The general pattern is also in this test

https://github.com/DataDog/dd-trace-py/blob/58249cfe2f7921eebb9e2712ff73308c83bb38f9/tests/internal/test_forksafe.py#L299